examples/Migration/Program.cs

```diff
@@ -0,0 +1,98 @@
+            // and Google Ads API.
+            string developerToken = "INSERT_YOUR_DEVELOPER_TOKEN_HERE";
+            long clientCustomerId = long.Parse("INSERT_YOUR_CLIENT_CUSTOMER_ID_HERE");
+            string oAuth2clientId = "INSERT_OAUTH2_CLIENT_ID_HERE";
+            string oAuth2Secret = "INSERT_OAUTH2_CLIENT_SECRET_HERE";
+            string oAuth2RefreshToken = "INSERT_OAUTH2_REFRESH_TOKEN_HERE";
+
+            // Login Customer ID is a new configuration setting required in Google Ads API. This
+            // is the customer ID of the authorized customer to use in the request, without
+            // hyphens(-).If your access to the customer account is through a manager account,
+            //this header is required and must be set to the customer ID of the manager account.
+            // See https://developers.google.com/google-ads/api/docs/concepts/call-structure#login-customer-id
+            // to learn more about this setting.
+            long loginCustomerId = long.Parse("INSERT_LOGIN_CUSTOMER_ID_HERE");
+
+            // Initialize logging. Since this code example uses the same log file for both APIs,
+            // you need to initialize the log stream once and reuse it for both libraries.
+            StreamWriter writer = new StreamWriter("C:\\logs\\details.log");
+            gacu::TraceUtilities.Configure(gacu::TraceUtilities.DETAILED_REQUEST_LOGS_SOURCE,
+                writer, System.Diagnostics.SourceLevels.All);
+
+            gagu::TraceUtilities.Configure(gagu::TraceUtilities.DETAILED_REQUEST_LOGS_SOURCE,
+                writer, System.Diagnostics.SourceLevels.All);
+
+            // Create AdWordsAppConfig and AdWordsUser instances, to be used for AdWords API calls.
+            AdWordsAppConfig adWordsAppConfig = new AdWordsAppConfig()
+            {
+                DeveloperToken = developerToken,
+                ClientCustomerId = clientCustomerId.ToString(),
+                OAuth2ClientId = oAuth2clientId,
+                OAuth2ClientSecret = oAuth2Secret,
+                OAuth2RefreshToken = oAuth2RefreshToken
+            };
+
+            AdWordsUser adWordsUser = new AdWordsUser(adWordsAppConfig);
+
+            // Create GoogleAdsConfig and GoogleAdsClient instances, to be used for AdWords API
+            // calls.
+            GoogleAdsConfig googleAdsConfig = new GoogleAdsConfig()
+            {
+                DeveloperToken = developerToken,
+                LoginCustomerId = loginCustomerId.ToString(),
+                OAuth2ClientId = oAuth2clientId,
+                OAuth2ClientSecret = oAuth2Secret,
+                OAuth2RefreshToken = oAuth2RefreshToken,
+            };
+
+            GoogleAdsClient googleAdsClient = new GoogleAdsClient(googleAdsConfig);
+
+            // Uncomment the relevant code example to run it.
+
+            // CampaignManagement example.
+            // new Step0().Run(adWordsUser);
```

Comment:
Can we rename these? When I personally see steps, I feel the need to run them all together. I believe the goal is to only run one of these at any point in time. So, here are my suggestions:

In the comment, recommend only one of these should be commented in and that the goal is to migrate from using the first one in the list to the last one in the list.

As for the naming of the classes, can we use Stage or Phase instead of Step? Also, it would help to quickly see if we gave them names. Here are my thoughts:
* Step0 => CreateCompleteCampaignAdWordsApiOnly
* Step1 => CreateCompleteCampaignBothApisPhase1
* Step2 => CreateCompleteCampaignBothApisPhase2
* Step3 => CreateCompleteCampaignBothApisPhase3
* Step4 => CreateCompleteCampaignBothApisPhase4
* Step5 => CreateCompleteCampaignGoogleAdsApiOnly